sklearn/cross_decomposition/_pls.py

```diff
@@ -23,6 +23,21 @@
 __all__ = ['PLSCanonical', 'PLSRegression', 'PLSSVD']
 
 
+def _pinv2_old(a):
+    # Used previous scipy pinv2 that was updated in:
+    # https://github.com/scipy/scipy/pull/10067
```

Comment:
Maybe make it explicit that it will be possible to re-switch back to scipy implementation by passing a suitable `cond` parameter once we stop supporting scipy < 1.3.

Reply:
EDIT: not possible as long as relative cutoff is not supported in recent scipy versions as explained below.